library/ix-dev/community/mumble/questions.yaml

```diff
@@ -0,0 +1,189 @@
+groups:
+  - name: Mumble Configuration
+    description: Configure Mumble
+  - name: Network Configuration
+    description: Configure Network for Mumble
+  - name: Storage Configuration
+    description: Configure Storage for Mumble
+  - name: Resources Configuration
+    description: Configure Resources for Mumble
+
+questions:
+  - variable: mumbleConfig
+    label: ""
+    group: Mumble Configuration
+    schema:
+      type: dict
+      attrs:
+        - variable: serverPassword
+          label: Server Password
+          description: The password for the Mumble Server.
+          schema:
+            type: string
+            default: ''
```

Comment:
There is no need to provide an empty string here and `empty` should be false as this has been provided and will pass validation

Reply:
Same as below